python/chronos/src/bigdl/chronos/forecaster/autoformer_forecaster.py

```diff
@@ -448,3 +531,18 @@ def metric(y_label, y_predict):
             return metric_func(y_label, y_predict)
         metric.__name__ = metric_name
     return metric
+
+
+def _timedelta_to_delta_str(offset):
+    features_by_offsets = (
+        (Timedelta(seconds=60), 's'),  # 6 for second - minutes
+        (Timedelta(minutes=60), 't'),  # 5 for minutes - hour
+        (Timedelta(hours=24), 'h'),  # 4 for hour - day
+        (Timedelta(days=7), 'd'),  # 3 for day - week
+        (Timedelta(days=30), 'w'),  # 2 for week - month
+        (Timedelta(days=365), 'm'),  # 1 for month - year
```

Comment:
what does these number comment( "1/2/3/4/5/6") mean ? Don't find where use these info.

Reply:
I will delete these, the number indicates how many features will be generated in x_time and y_time under this freq